contents/spotlight/startup-alignwith.md

```diff
@@ -0,0 +1,57 @@
+
+“At my old companies we make every mistake you could possibly make,” says Kenan. “And by going through those issue and feeling the pain of scaling we realized how important it was to keep people aligned with our product teams. That led to us creating decks, release notes, training materials...and eventually, the Alignwith platform.”
+
+Through tools such as AI documentation Alignwith offers a better way for product teams to share their ideas — but the company’s brand is just as impressive and unique as the technology underneath it. So, we spoke to Kenan about how he built a brand which users love. 
+
+## 1. You can be anything, so be ~a wolf~ unique
+
+A former artist, musician, and photographer in the vibrant New York scene, Kenan says the first step to building anything should be a familiarity with what the status quo already is. Only then can you build something that defies the norm and stands out in a crowded market.
+
+“I think that’s like an artist’s operating system,” says Kenan. “You always want your work to stand out and that means always looking at what everyone else is making so you can get inspired by it or go in different directions.”
+
+In the case of Alignwith Kenan wanted to stand out against the [corporate memphis](https://en.wikipedia.org/wiki/Corporate_Memphis) style seen on many other startup websites. He wanted to defy the idea of small-headed humans in pastel primitives and present something different: bright, hand-drawn wolves in business suits as an embodiment of the brand. 
+
+![Alignwith mascots](https://res.cloudinary.com/dmukukwp6/image/upload/v1713973218/posthog.com/contents/images/blog/alignwith-wolf.png)
+
+## 2. Take your time, do it yourself
+
+In a time when AI is becoming ubiquitous and freelance marketplaces make it effortless to enlist temporary employees, is it still worth recruiting full-time staff? In our experience the answer is [yes, absolutely](/founders/posthog-first-five#4-lottie-coxon) — and Kenan agrees. 
+
+“We’ve tried AI tools and we didn’t like the results we got,” says Kenan. “We even tried one where we uploaded our whole design system and...honestly, you couldn’t pay me to implement those ideas.”
+
+“People’s perception of your team and their desire to use your software really pivots on whether it’s a good design. It’s not worth settling for ‘good enough’, it’s gotta be _great_ enough. For that, you need to find a good designer to work with.”
+
+“My designer is very patient and that’s important because I’m always picking every little thing apart until I think it’s ready!”
+
+## 3. Bring your brand into your product
+
+It’s not enough to simply design a mascot and apply a color palette, says Kenan. You need to create an entire personality around your brand and then, ideally, extend it into your product so that users touch it whenever they interact with you.
```

Comment:
```suggestion
It’s not enough to simply design a mascot and apply a color palette, says Kenan. You need to create an entire personality around your brand. Then, ideally, you can extend it into your product so that users touch it whenever they interact with you.
```